src/i18n/languages/de-CH.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Stefan Salzl
+ * Last updated: Dec 12, 2017
+ *
+ * Description: Definition file for German - Switzerland language-country.
+ */
+import * as C from '../constants';
+
+const dictionary = {
+  languageCode: 'de-CH',
+  [C.CONTEXTMENU_ITEMS_ROW_ABOVE]: 'Zeile einfügen oberhalb',
+  [C.CONTEXTMENU_ITEMS_ROW_BELOW]: 'Zeile einfügen unterhalb',
+  [C.CONTEXTMENU_ITEMS_INSERT_LEFT]: 'Spalte einfügen links',
+  [C.CONTEXTMENU_ITEMS_INSERT_RIGHT]: 'Spalte einfügen rechts',
+  [C.CONTEXTMENU_ITEMS_REMOVE_ROW]: ['Zeile löschen', 'Zeilen löschen'],
+  [C.CONTEXTMENU_ITEMS_REMOVE_COLUMN]: ['Spalte löschen', 'Spalten löschen'],
+  [C.CONTEXTMENU_ITEMS_UNDO]: 'Rückgangig',
+  [C.CONTEXTMENU_ITEMS_REDO]: 'Wiederholen',
+  [C.CONTEXTMENU_ITEMS_READ_ONLY]: 'Nur Lesezugriff',
+  [C.CONTEXTMENU_ITEMS_CLEAR_COLUMN]: 'Spalteninhalt löschen',
+
+  [C.CONTEXTMENU_ITEMS_ALIGNMENT]: 'Ausrichtung',
+  [C.CONTEXTMENU_ITEMS_ALIGNMENT_LEFT]: 'Linksbündig',
+  [C.CONTEXTMENU_ITEMS_ALIGNMENT_CENTER]: 'Zentriert',
+  [C.CONTEXTMENU_ITEMS_ALIGNMENT_RIGHT]: 'Rechtsbündig',
+  [C.CONTEXTMENU_ITEMS_ALIGNMENT_JUSTIFY]: 'Blocksatz',
+  [C.CONTEXTMENU_ITEMS_ALIGNMENT_TOP]: 'Oben',
+  [C.CONTEXTMENU_ITEMS_ALIGNMENT_MIDDLE]: 'Zentriert',
+  [C.CONTEXTMENU_ITEMS_ALIGNMENT_BOTTOM]: 'Unten',
+
+  [C.CONTEXTMENU_ITEMS_FREEZE_COLUMN]: 'Spalte fixieren',
+  [C.CONTEXTMENU_ITEMS_UNFREEZE_COLUMN]: 'Spaltenfixierung aufheben',
+
+  [C.CONTEXTMENU_ITEMS_BORDERS]: 'Rahmen',
+  [C.CONTEXTMENU_ITEMS_BORDERS_TOP]: 'Oben',
+  [C.CONTEXTMENU_ITEMS_BORDERS_RIGHT]: 'Rechts',
+  [C.CONTEXTMENU_ITEMS_BORDERS_BOTTOM]: 'Unten',
+  [C.CONTEXTMENU_ITEMS_BORDERS_LEFT]: 'Links',
+  [C.CONTEXTMENU_ITEMS_REMOVE_BORDERS]: 'Kein Rahmen',
+
+  [C.CONTEXTMENU_ITEMS_ADD_COMMENT]: 'Kommentar hinzufügen',
+  [C.CONTEXTMENU_ITEMS_EDIT_COMMENT]: 'Kommentar bearbeiten',
+  [C.CONTEXTMENU_ITEMS_REMOVE_COMMENT]: 'Kommentar löschen',
+  [C.CONTEXTMENU_ITEMS_READ_ONLY_COMMENT]: 'Kommentar sperren',

```

Comment:
I believe `Kommentar sperren` may be too abstract, because it doesn't tell the user that he is unable to change the comment.

Reply:
Thank you for your suggestion @madoar 
Is there anything else that would fit here better?

Reply:
It depends on your length requirements.
I believe something like `Kommentarbearbeitung sperren` or `Bearbeitung von Kommentaren sperren` (which both translate to: `lock comment changes`) would fit better, but these texts are quite long.

Reply:
Hi @madoar 
we will be expanding the context menu a little bit. As @wszymanski wrote at https://github.com/handsontable/handsontable/issues/4657 it will have `215px`

Reply:
@madoar @AMBudnik I tried to make the translation as close as possible to the current German Excel Version. That's why I went with those three (Excel's naming is not very consistent). 

My proposed changes would be:
1. Leave `[C.CONTEXTMENU_ITEMS_ALIGNMENT_CENTER]: 'Zentriert'` <-- That's consistent across all German Office Programs (be it MS or Libre etc.)
2. Change `[C.CONTEXTMENU_ITEMS_ALIGNMENT_MIDDLE]: 'Mitte'` <-- That will fit with "Oben" and "Unten" & it is at least consistent with Powerpoint's naming convention
3. Change `[C.CONTEXTMENU_ITEMS_READ_ONLY_COMMENT]: 'Schreibschutz Kommentar'`, roughly translates to `"Writeprotection Comment"`, because it is slightly shorter than the alternative.

Reply:
@buggythepirate sounds good to me :)
@AMBudnik about the width: 
I've discovered that menu item labels don't automatically use line breaks, if the label is longer than the width, but instead are cut off. Is it planned to change this behavior?

Reply:
Hi @madoar 
The contextMenu topic is something different and we need to keep this conversation translation-strict but I think that the menu should do 'something' when the title is too long. We can create a new issue to discuss some alternatives.